src/main/java/com/oldoe/plugin/Oldoe.java

```diff
@@ -33,18 +35,19 @@ public static Oldoe getInstance() {
 
     @Override
     public void onEnable() {
-
         instance = this;
         initConfig();
 
         GetServiceManager().Register(instance);
 
         this.softBlocks = GetSoftBlocksMaterials();
+        oldoeMute.loadMutedPlayers();
```

Comment:
We can add this as a column in the User DB, I would prefer that over adding an additional flatfile